concepts/language-model.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Language model
+---
+
+# Language model
+
+A language model tries to predict the next word given the already produced history.
+For example, given the text span `the man with` a language model's output could be `a: 0.4, the: 0.3, my: 0.1, his: 0.05, ...`.
+This would mean that the article `a` has a 40% probability of being the continuation and is the most likely.
```

Comment:
> In this example, the model predicts that the word `bike` has the highest probability (40%) of being the next word.

We could even use % in the example above.

Reply:
Good point, changed.